ios/FluentUI.Demo/FluentUI.Demo/Demos/ListActionItemDemoController_SwiftUI.swift

```diff
@@ -80,58 +80,69 @@ struct ListActionItemDemoView: View {
-                        .backgroundStyleType(backgroundStyleType)
-                    }
-
-                } header: {
-                    Text("ListActionItem")
-                }
-                .alert("Action tapped", isPresented: $showingAlert) {
-                    Button("OK", role: .cancel) { }
-                        .accessibilityIdentifier("DismissAlertButton")
+            FluentListSection {
+                if showSecondaryAction {
+                    ListActionItem(primaryActionTitle: primaryActionTitle,
+                                   onPrimaryActionTapped: {
+                        showingAlert.toggle()
+                    },
+                                   primaryActionType: primaryActionType,
+                                   secondaryActionTitle: secondaryActionTitle,
+                                   onSecondaryActionTapped: {
+                        showingAlert.toggle()
+                    },
+                                   secondaryActionType: secondaryActionType)
+                    .topSeparatorType(topSeparatorType)
+                    .bottomSeparatorType(bottomSeparatorType)
+                    .backgroundStyleType(backgroundStyleType)
+                } else {
+                    ListActionItem(title: primaryActionTitle,
+                                   onTapped: {
+                        showingAlert.toggle()
+                    },
+                                   actionType: primaryActionType)
+                    .topSeparatorType(topSeparatorType)
+                    .bottomSeparatorType(bottomSeparatorType)
+                    .backgroundStyleType(backgroundStyleType)
                 }
+            } header: {
+                Text("ListActionItem")
+            }
+            .alert("Action tapped", isPresented: $showingAlert) {
+                Button("OK", role: .cancel) { }
+                    .accessibilityIdentifier("DismissAlertButton")
+            }
 
-                Section {
-                    FluentUIDemoToggle(titleKey: "Show secondary action", isOn: $showSecondaryAction)
-                        .accessibilityIdentifier("showSecondaryActionSwitch")
-                    textFields
-                    pickers
-                } header: {
-                    Text("Settings")
-                }
+            FluentListSection {
+                FluentUIDemoToggle(titleKey: "Show secondary action", isOn: $showSecondaryAction)
+                    .accessibilityIdentifier("showSecondaryActionSwitch")
+                textFields
+                pickers
+            } header: {
+                Text("Settings")
             }
-            .fluentTheme(fluentTheme)
-            .listStyle(.insetGrouped)
         }
 
-        return content
+        return FluentList {
+            content
+        }
+        .fluentListStyle(listStyle)
+        .background(ListItem.listBackgroundColor(for: .grouped))
+        .fluentTheme(fluentTheme)
```

Comment:
nit: can we move this to a viewbuilder and just have return contentView or something like that